### PR TITLE
Implemented basic UI functionality using plain HTML for the Awesome Books project

### PR DESCRIPTION
**Pull request by ahmidou09**
This pull request introduces the basic user interface for the Awesome Books project. The project aims to create a website where users can add and remove books from a list using JavaScript objects and arrays. The following features have been implemented:

1. Creation of a collection: An array of objects has been created to serve as the collection that keeps a list of books.

2. Addition of a book: A function has been implemented to add a new book to the collection. The function takes the title and author as parameters and appends a new object representing the book to the array.

3. Removal of a book: Another function has been implemented to remove a book from the collection. The function uses the `filter()` method on the array to exclude the book that matches the specified criteria.

4. Display of all books: The top part of the page now displays all the books saved in the collection. This is achieved by iterating through the array and dynamically generating the necessary HTML elements to display each book's title and author.

5. Data persistence using localStorage: To ensure that data is preserved in the browser's memory, the implementation utilizes the localStorage feature provided by the browser. The book collection is stored in the localStorage, allowing the data to persist even if the page is refreshed or closed.

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ahmidou09 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that has no README file. Kindly add the [accepted Microverse readme.](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo <a name="live-demo"></a>
+
+🚀 Live Demo
+Link to our deployed project.
+
+- https://ahmidou09.github.io/Awesome-books.git
+
```
Comment (Lizdev-05) [inline README.md]: Great job adding the README file, kindly remove the link or update it as it is directing to an error 404 🥺

File: index.html [hunk 2]
```diff
@@ -3,7 +3,29 @@
...
-  <body></body>
+  <body>
+    <div class="container">
+      <h1>Awesome books</h1>
+      <div class="listBooks">
+        <!--  <ul class="book">
+          <li>Title: azz</li>
+          <li>Author: zazaz</li>
+          <button class="removeBook">remove</button>
+        </ul> -->
```
Comment (Lizdev-05) [inline index.html]: Kindly remove the commented code so as to make your code base neat 🙏 .

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hello @ahmidou09 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!


## Highlights

-  Descriptive Pull Request message ✔️
-  Good use of GitHub flow ✔️
-  Meaningful commit messages ✔️
-  No linter error ✔️
-  Brilliant implementation ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @ahmidou09  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**